apps/vocabulary/vocabulary.pb.go

```diff
@@ -0,0 +1,268 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Please move this file and `vocabulary.proto` to a new top-level directory in the gnostic repo called `metrics`. 

To regenerate the file, please add this line to the end of `COMPILE-PROTOS.sh` at the top-level:

```protoc --go_out=. metrics/vocabulary.proto```

Then you can regenerate the file and import it from `apps/vocabulary/main.go`